### ISSUE TITLE
Unused parameter in closure should be replaced with _

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#982]
Unused parameter in a closure should be replaced with `_`:

```swift
// should trigger
[1, 2].map { number in
    return 3
}

// ok
[1, 2].map { _ in
    return 3
}

[1, 2].map { number in
    return number + 1
}

[1, 2].map { $0 + 1 }
```
